docs/csharp/language-reference/operators/index.md

```diff
@@ -10,11 +11,45 @@ helpviewer_keywords:
+C# provides a number of operators. Many of them are supported by the [built-in types](../builtin-types/built-in-types.md) and allow you to perform basic operations with values of those types. Those operators include the following groups:
 
-In an [expression](../../programming-guide/statements-expressions-operators/expressions.md), operator precedence and associativity determine the order in which the operations are performed. You can use parentheses to change the order of evaluation imposed by operator precedence and associativity.
+- [Arithmetic operators](arithmetic-operators.md) that perform arithmetic operations with numeric operands
+- [Comparison operators](comparison-operators.md) that compare numeric operands
+- [Boolean logical operators](boolean-logical-operators.md) that perform logical operations with [`bool`](../builtin-types/bool.md) operands
+- [Bitwise and shift operators](bitwise-and-shift-operators.md) that perform bitwise or shift operations with operands of the integral types
+- [Equality operators](equality-operators.md) that check if their operands are equal or not
+
+Typically, you can [overload](operator-overloading.md) those operators, that is, specify the operator behavior for the operands of a user-defined type.
+
+The simplest C# expressions are literals (for example, [integer](../builtin-types/integral-numeric-types.md#integer-literals) and [real](../builtin-types/floating-point-numeric-types.md#real-literals) numbers) and names of variables. You can combine them into complex expressions by using operators. Operator [precedence](#operator-precedence) and [associativity](#operator-associativity) determine the order in which the operations in an expression are performed. You can use parentheses to change the order of evaluation imposed by operator precedence and associativity.
+
+In the following code, examples of expressions are at the right-hand side of assignments:
+
+[!code-csharp[expression examples](snippets/Overview.cs#Expressions)]
+
+Typically, an expression produces a result and can be included into another expression. A [`void`](../builtin-types/void.md) method call is an example of an expression that doesn't produce a result. It can be used only as a [statement](../../programming-guide/statements-expressions-operators/statements.md), as the following example shows:
+
+```csharp
+Console.WriteLine("Hello, world!");
+```
+
+Here are some other kinds of expressions that C# provides:
+
+- [Interpolated string expressions](../tokens/interpolated.md) that provide convenient syntax to create formatted strings:
+
+  [!code-csharp-interactive[interpolated string](snippets/Overview.cs#InterpolatedString)]
```

Comment:
If you used the new syntax here, I believe you could highlight the line(s) referred to in the text, i.e., in this case the `var message =` line.

Reply:
If highlighting by line number will be added, an *access-by-line.txt* file will be needed.

Reply:
@tdykstra thanks for a suggestion, but I'll leave it as it is now: the snippet is really small.